.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: CI
```

Comment:
We just added CI.yml for this. Can we work with that to not create duplicates? Thanks.